packages/systemd/9007-Revert-pkg-config-prefix-is-not-really-configurable-.patch

```diff
@@ -0,0 +1,84 @@
+From ed49352fa7cf7d9c2e371849dca3ebf906e800bc Mon Sep 17 00:00:00 2001
+From: Erikson Tung <etung@amazon.com>
+Date: Mon, 11 Jan 2021 11:01:57 -0800
+Subject: [PATCH] Revert "pkg-config: prefix is not really configurable, don't
```

Comment:
It looks like we could get what we need from upstream's revert rather than doing it ourselves, possibly with the addition of a small `prefix=@prefix@` patch, if that's needed.  https://github.com/systemd/systemd/commit/60bce7c6d9606185114df1bdcd5ea100407688b8

Reply:
We still need the `prefix=@prefix@`. It doesn't work with `prefix` hardcoded to `/usr`.

I'll go ahead and pick that commit and add a small patch.